### ISSUE TITLE
ReferenceError: window is not defined

### ISSUE THREAD

**Issue by ramilexe** [transmute-industries/transmute#159]
```
/usr/src/app/node_modules/transmute-framework/src/event-store/index.js:58
    if (window.web3) {
    ^

ReferenceError: window is not defined
    at new EventStore (/usr/src/app/node_modules/transmute-framework/src/event-store/index.js:58:5)
    at Object.<anonymous> (/usr/src/app/packages/users-manager/index.js:15:20)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/usr/src/app/routes/users.js:6:21)
```

```
ramil@-api$ cat package.json | grep transmute
    "transmute-framework": "^0.2.0-alpha.2",
```

**Comment by OR13** [comment]
thanks, this is caused by adding metamask support incorrectly.

**Comment by OR13** [comment]
will hotfix shortly

**Comment by OR13** [comment]
This has been resolved in `transmute-framework@0.2.3`
